core/templates/dev/head/pages/exploration-editor-page/improvements-tab/templates/answer-details-modal.template.html

```diff
@@ -10,15 +10,15 @@ <h3 class="modal-title">
   <div class="modal-body" style="margin-left:0.5rem; margin-right: 0.5rem;">
     <table class="table table-hover">
       <tr ng-if="getLearnerAnswerInfos().length > 0">
-        <th scope="col">Select</th>
+        <th scope="col" ng-if="isUserLoggedIn">Select</th>
```

Comment:
@brianrodri, is this should only visible to the users with the edit rights? (Ditto below!)

Reply:
I'm not sure about the expected behavior, but I think the components of the editor page should be visible to all (including guest users) and only the editors/owners have the edit rights!

Reply:
This simply hides the components that would allow a user to make edits, as per @seanlip's suggestion of following the exploration editor's example of only exposing elements when they are actually usable by the creator.

Reply:
So I don't think this would be usable for a user who doesn't have edit rights on exploration, right?

Reply:
Done